From 948525fd0ca553d4fce84fc5b080278494c4f224 Mon Sep 17 00:00:00 2001
From: Marco Sirabella <marco@sirabella.org>
Date: Tue, 18 May 2021 17:43:35 -0400
Subject: [PATCH] Search for the library containing: tigetnum (tinfo)

For me, `tigetnum` is found in tinfo.so, ncurses has been split off into
tinfo due to being compiled with the --with-termlib flag.

pkg-config --libs automatically reports this, but this check is good
enough to catch this specific problem.
---
 configure.ac | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configure.ac b/configure.ac
index b617f9b1..b892ffea 100644
--- a/configure.ac
+++ b/configure.ac
@@ -113,6 +113,7 @@ AC_CHECK_FUNCS([geteuid getpwuid fmemopen strsep])
 AC_SEARCH_LIBS([endwin],[curses ncurses])
 AC_SEARCH_LIBS([readline],[readline])
 AC_SEARCH_LIBS([add_history],[readline history])
+AC_SEARCH_LIBS([tigetnum],[curses ncurses tinfo])
 
 # If readline wasn't disabled by the user, does it actually exist and is it a
 # proper readline?
